src/velocity.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+from scipy.interpolate import interp2d
+
+def interpolated_value(lat, long, vel_array, xy_positions, step = 0.01):
+    """Returns the interpolated value of the tensor for the latitude and longitude requested.
+    
+    """
```

Comment:
Añado la explicación de algunos parámetros.

```suggestion
    """
    Returns the interpolated value of the tensor for the latitude and longitude requested.
    Params
    --------
    lat : float
        Position of the boat (latitude)
    long : float
        Position of the boat (longitude)
    vel_array : numpy.array
        The velocity (v_x, v_y) of the stream at each point.
        shape = (lat, lon, 2)
    xy_positions : tuple of numpy.array
       Positions of each stream. For instance,
       stream_velocitites[i, j, :] corresponds to position
       ( xy_stream_velocities[0][i], xy_stream_velocities[1][j] )
    step : float, default=0.01
    """
```